### PR TITLE
Dynamic login

### PR DESCRIPTION
**Pull request by hsammon2**
This is the base code for the login functionality, log out has also been added but the button to preform this action needs to be added to the nav bar. I couldn't figure that out......

### CODE REVIEW

File: templates/login.html [hunk 1]
```diff
@@ -1,12 +1,22 @@
 {% extends '_layout.html' %}
 
+{% block body %}
+<h2 class="mb-5">Login to Your account</h2>
```
Comment (tylerminnis) [inline templates/login.html:4]: lowercase for headings (as long as we're still doing this i'm not entirely sure)

File: templates/register.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{% extends '_layout.html' %}
+
+{% block body %}
+<h2 class="mb-5">Login to Your account</h2>
```
Comment (tylerminnis) [inline templates/register.html:4]: same thing as the other heading

**Review by tylerminnis — APPROVED** [review]
sounds good, comments only regard lower case headings but at the same time idk if we're doing that. 
also, why does github say u touched every single file in the code when in reality u didn't? idk 

LGTM

**Comment by tylerminnis** [comment]
i'm sorry did that say 'add files via upload'... why do u use the gui :(

**Comment by Nathan-Laney** [comment]
lgtm (let's get this merge)
also i hate github gui

**Review by Nathan-Laney — APPROVED** [review]

File: templates/register.html (hunk 2 above)
Comment (sumiv) [reply templates/register.html:4]: is it just headings or do we want all text fields (like First Name, Email) to be lowercase as well?

Comment (tylerminnis) [reply templates/register.html:4]: idk i meant to talk abt that last week but we'll talk abt it during next sprint or retro

**Pull request merged**